packages/react/error-boundary/src/index.ts

```diff
@@ -1,3 +1,4 @@
 /** @tossdocs-ignore */
-export { default as ErrorBoundary } from './ErrorBoundary';
+export { ResetKey, useResetKey, withResetKey } from './contexts/ResetKey';
```

Comment:
As noted in the earlier comment, I think 

- `ErrorBoundaryContext` & `withErrorBoundaryContext`
- `ErrorBoundaryGroup` & `withErrorBoundaryGroup`

might be a better naming.

Reply:
In case of `useResetKey`, what about `useErrorBoundaryContext` ?

```tsx
const context = useErrorBoundaryContext();

context.reset();
```

I think `resetKey` is implementation detail and I would like to hide it from the public API.